Header header
float32 Batt_Voltage
bool RC_OK
bool Motors_ON
bool ExCtrl_ON
duration Run_Time
